### DISCUSSION TITLE
Privacy Manifest

### DISCUSSION THREAD

**Discussion by EricJMoon** [krzyzanowskim/CryptoSwift#1034] [Q&A] [answered]
Hi,
At WWDC23 Apple announced that apps and SDKs that make use of "required reason" APIs and "Data Collection", "Tracking" will need to provide a privacy manifest.
Does CryptoSwift need to include this manifest?

https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests
https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api

**Comment by krzyzanowskim** [comment, marked as answer]
> Does CryptoSwift need to include this manifest?

yes. the technical side of it is not entirely clear for me for the SPM distributed project. That is something on my rada already.

**Reply by EricJMoon** [reply]
Thank you for your quick reply!!!

**Comment by krzyzanowskim** [comment]
Added in 1.8.1

**Comment by JLLA113** [comment]
Hi, 

This was closed. But I have a doubt.

Apple also mentioned that sdk's need to be signed by the developer. 

Ref: https://developer.apple.com/support/third-party-SDK-requirements/

Does this need to be adopted by the library or is already supported?

Thanks

**Comment by krzyzanowskim** [comment]
that only apply to binary distribution
